test/integration/vaultauthmethods_integration_test.go

```diff
@@ -71,6 +75,14 @@ func TestVaultAuthMethods(t *testing.T) {
 	if r := os.Getenv("AWS_REGION"); r != "" {
 		awsRegion = r
 	}
+	runGCPTests := true
+	if run, _ := runGCP(t); !run {
+		runGCPTests = false
+	}
+	gcpRegion := defaultGCPRegion
+	if r := os.Getenv("GCP_REGION"); r != "" {
+		gcpRegion = r
```

Comment:
> Is there a reason we are not getting the GCP project as well?
Just that there's no default for the project. I went ahead and removed this block since setting a default region on this end didn't seem necessary.